SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -247,9 +247,9 @@ SCDocHTMLRenderer {
 		<< "</script>\n"
 		<< "<script src='" << baseDir << "/scdoc.js' type='text/javascript'></script>\n"
 		<< "<script src='" << baseDir << "/docmap.js' type='text/javascript'></script>\n" // FIXME: remove?
-		<< "<script src='" << baseDir << "/frontend.js' type='text/javascript'></script>\n"
+		// << "<script src='" << baseDir << "/frontend.js' type='text/javascript'></script>\n" // does not exist
 		// QWebChannel access
-		<< "<script src='qrc:///qtwebchannel/qwebchannel.js' type='text/javascript'></script>\n"
+		// << "<script src='qrc:///qtwebchannel/qwebchannel.js' type='text/javascript'></script>\n" // does not exist
```

Comment:
This exists within the Qt IDE as far as I am aware but I don't know for what, so its best to keep it in place.

Reply:
Thanks for letting me know it!